src/snowplow/lib/base64.lua

```diff
@@ -47,141 +47,199 @@
 -- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 -- THE SOFTWARE.
 
-require "lib.bit"
+require("lib.bit")
 
 base64 = {}
- 
+
 --- octet -> char encoding.
 local ENCODABET = {
-	'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J',
-	'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T',
-	'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd',
-	'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n',
-	'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x',
-	'y', 'z', '0', '1', '2', '3', '4', '5', '6', '7',
-	'8', '9', '-', '_'
+	"A",
```

Comment:
This is a bit unnecessary to list each character on a new line but I understand that it's probably the output of the formatter, so no problem.